commands/flags.go

```diff
@@ -2,21 +2,31 @@ package commands
 
 import (
 	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+
 	"github.com/gruntwork-io/health-checker/options"
 	"github.com/gruntwork-io/gruntwork-cli/logging"
 	"github.com/urfave/cli"
 	"github.com/sirupsen/logrus"
-	"os"
-	"strings"
 )
 
+const DEFAULT_CHECKS_FILE = "health-checks.yml"
 const DEFAULT_LISTENER_IP_ADDRESS = "0.0.0.0"
 const DEFAULT_LISTENER_PORT = 5500
 const ENV_VAR_NAME_DEBUG_MODE = "HEALTH_CHECKER_DEBUG"
 
-var portFlag = cli.IntSliceFlag{
-	Name: "port",
-	Usage: fmt.Sprintf("[Required] The port number on which a TCP connection will be attempted. Specify one or more times. Example: 8000"),
+type Checks struct {
+	Ports	[]int	  `yaml:"ports"`
+}
+
+var checksFlag = cli.StringFlag{
+	Name: "checks",
```

Comment:
`config`?

Reply:
So I renamed this as I think calling it checks makes more sense - I'm OK with going back to config - but I actually left out "configs" like the listener or log level for example...  those still get set via the cli args